pvlib/pvsystem.py

```diff
@@ -1887,6 +1887,82 @@ def sapm_celltemp(poa_global, wind_speed, temp_air,
     return pd.DataFrame({'temp_cell': temp_cell, 'temp_module': temp_module})
 
 
+def pvsyst_celltemp(
+    poa_global,
+    wind_speed,
+    temp_air,
+    eta_m=0.1,
+    alpha_absorption=0.9,
+    temp_model="freestanding",
+):
+    """
+    Calculate PVSyst cell temperature.
+
+    Parameters
+    ------------
```

Comment:
I can't remember if sphinx will complain about the two extra `-`. Might as well remove the extra here, below Returns, and in api.rst while you're cleaning up the other items.

Reply:
Sorry, haven't done documentation in sphinx previously. Just to be sure, the rules are that there are as many dashes as characters in the word above it, correct?

Reply:
yes.